tests/cli/main.cpp

```diff
@@ -624,6 +624,753 @@ BOOST_FIXTURE_TEST_CASE( create_son, cli_fixture )
    BOOST_TEST_MESSAGE("SON cli wallet tests end");
 }
 
+BOOST_FIXTURE_TEST_CASE( select_top_fifteen_sons, cli_fixture )
+{
+   BOOST_TEST_MESSAGE("SON cli wallet tests begin");
+   try
+   {
+      INVOKE(upgrade_nathan_account);
+
+      graphene::wallet::brain_key_info bki;
+      signed_transaction create_tx;
+      signed_transaction transfer_tx;
+      signed_transaction upgrade_tx;
+      signed_transaction vote_tx;
+      account_object acc_before_upgrade, acc_after_upgrade;
+      son_object son_obj;
+      global_property_object gpo;
+
+      // create sonaccount01
```

Comment:
It would be great to move SON creation in separate function to make the code simpler,  but I think it can be refactored later if https://github.com/peerplays-network/peerplays/pull/179 will be merged